### ISSUE TITLE
Notifications don't trigger

### ISSUE THREAD

**Issue by theCalcaholic** [nextcloud/flow_notifications#15]
I have set up a simple test case:

![image](https://user-images.githubusercontent.com/6317548/107450499-a6c86b00-6b45-11eb-9dda-ef6e3bb01de7.png)

The NC Talk flow is just there to verify that I didn't misconfigure the conditions. When I edit/rename a file called `test.md` to `test1.md` (or vice versa), I get a message in Talk but no notification.

NC version: 20.0.4
Flow Notifications version: 1.0.3

**Comment by MarVinSto** [comment]
Can confirm with:
NC version: 20.0.5
Flow Notifications version: 1.0.3

**Comment by snarchio** [comment]
Can confirm with:
NC version: 20.0.8
Flow Notifications version: 1.0.3

**Comment by blizzz** [comment]
Cannot confirm on 20.0.8 / 1.0.3.

Please note that the notification is not triggered, when you do the action yourself.

**Comment by snarchio** [comment]
> Cannot confirm on 20.0.8 / 1.0.3.
> 
> Please note that the notification is not triggered, when you do the action yourself.

Sorry, my mistake in testing after the update

**Comment by theCalcaholic** [comment]
@blizzz Oh, so this is intended behavior? IMO, this could be configurable. I don't always access files manually, but also in various automated tasks via scripts, which might be a use case for this flow.

Anyways, I'll test the app with another user and report back in a minute.

**Comment by blizzz** [comment]
> @blizzz Oh, so this is intended behavior? IMO, this could be configurable. I don't always access files manually, but also in various automated tasks via scripts, which might be a use case for this flow.
> 
> Anyways, I'll test the app with another user and report back in a minute.

Yes, it is intended and also worked like this. Does not mean it has to be like this forever (but then opt-in).

**Comment by MarVinSto** [comment]
> Cannot confirm on 20.0.8 / 1.0.3.
> 
> Please note that the notification is not triggered, when you do the action yourself.

Sorry, this was my fault!
Is it possible that notifications are also not created if the files are in group folders?
